index.src.html

```diff
@@ -519,14 +520,18 @@ <h3 id="fetch-integration">Fetch Integration</h3>
   execute the following step:
 
   9.  If |response|'s <a>header list</a> contains a header named
-      <a>`Clear-Site-Data`</a>, then execute [[#clear-response]] on
-      |response|.
+      <a>`Clear-Site-Data`</a> and the request's <a>credentials mode</a> permits
+      modifying credentials, then execute [[#clear-response]] on |response|.
```

Comment:
I don't really see why the entire header is conditional upon this. By doing that we extend "credentials mode" to also affect things it didn't previously affect, such as `localStorage` and co. I guess it makes some sense to do that, but I'm a little wary about it.